core/src/main/proto/spine/core/diagnostics.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package spine.core;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.core";
+option java_outer_classname = "DiagnosticsProto";
+option java_multiple_files = true;
+
+import "google/protobuf/any.proto";
+
+import "spine/core/actor_context.proto";
+import "spine/core/version.proto";
+
+// An identifier of a message dispatched to an entity.
+//
+// May represent either an event or a command.
+//
+message MessageQualifier {
```

Comment:
Let's discuss this message and the `EntityQualifier` one. I don't think they are named according to what they are.